app/models/order_product.rb

```diff
@@ -0,0 +1,4 @@
+class OrderProduct < ApplicationRecord
```

Comment:
I would validate here that the price is present and quantity has to be a positive number